Training2/src/src/task1/Modal.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import './modal.css'
+
+const Modal = ({ isShowing, hide, children }) =>
+  isShowing ? (
```

Comment:
@doankhanh222 With standard modal, we should not remove modal dom and children in case we need animation for the modal for keep the modal DOM, so you should using style for hide it from UI